### PR TITLE
Reexport DataFrames

### LINKED ISSUE #185: Reexport DataFrames
It would probably make sense to reexport all of DataFrames, since you cannot do much with DataFramesMeta if you can't even use the `DataFrame` type.

### PR DESCRIPTION
**Pull request by pdeffebach**
fixes #185 

Less typing for me to get started also!

cc @nalimilan

### CODE REVIEW

**Review by bkamins — APPROVED** [review]

**Comment by bkamins** [comment]
you need to add Reexport.jl version in `[compat]` I think

**Comment by bkamins** [comment]
OK to merge when CI finishes.

**Comment by pdeffebach** [comment]
Thanks!

**Comment by pdeffebach** [comment]
All I did in this last commit is get rid of an unexported `DataFramesMeta.select` method which was used internally in the old macros, but not the new implementation.

File: src/DataFramesMeta.jl [hunk 1]
```diff
@@ -378,15 +380,6 @@ macro where(d, args...)
 end
 
 
-##############################################################################
-##
-## select - select columns
-##
-##############################################################################
-
-select(d::AbstractDataFrame, arg) = d[!, arg]
```
Comment (nalimilan) [inline src/DataFramesMeta.jl:387]: Wow, this one was... embarrassing. Thanks for the PR!

**Pull request merged**